CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 
 ### Module updates
 
+- **Picard**: fix parsing mixed strings/numbers, account for trailing tab ([#2083](https://github.com/ewels/MultiQC/pull/2083))
+
```

Comment:
Maybe a tweak needed for the changelog bot? Updates usually go on their own newlines under module name.
```suggestion
- **Picard**:
    - Fix parsing mixed strings/numbers, account for trailing tab ([#2083](https://github.com/ewels/MultiQC/pull/2083))

```